Libraries/LibWeb/HTML/HTMLHeadingElement.h

```diff
@@ -26,8 +26,12 @@ class HTMLHeadingElement final : public HTMLElement {
 
     virtual Optional<String> aria_level() const override
     {
-        // TODO: aria-level = the number in the element's tag name
-        return get_attribute("aria-level"_fly_string);
+        if (auto const attr = get_attribute("aria-level"_fly_string); attr.has_value())
```

Comment:
We create FlyStrings for standard attribute names already. For this, it'd be `ARIA::AttributeNames::aria_level`, in `LibWeb/ARIA/AttributeNames.h`.